userguide/SUMMARY.md

```diff
@@ -16,6 +16,7 @@ This is the User Guide for Turi Create
     * [Recommender systems](recommender/README.md)
         * [Using models](recommender/using-trained-models.md)
         * [Choosing a model](recommender/choosing-a-model.md)
+        * [Custom Recommender Model for iOS Applications](recommender/custom-model-ios.md)
```

Comment:
Simply call this export to CoreML (like all other user guide chapters)

Reply:
Note: they are called "Deployment to Core ML"